spec/models/manageiq/providers/embedded_ansible/automation_manager/job/status_spec.rb

```diff
@@ -1,3 +1,58 @@
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job::Status do
-  it_behaves_like 'ansible job status'
+  # The following specs are copied from the 'ansible job status' spec helper
+  # from the AnsibleTower Provider repo, but have been modified to make sense
+  # for the case of AnsibleRunner implementation.  Previously was:
+  #
+  # it_behaves_like 'ansible job status'
+
+  let(:miq_task) { MiqTask.new(:state => MiqTask::STATE_FINISHED) }
+
+  it 'parses Succeeded' do
+    miq_task.status = MiqTask::STATUS_OK
+    status          = described_class.new(miq_task, '')
+
+    expect(status.completed?).to   be_truthy
+    expect(status.succeeded?).to   be_truthy
+    expect(status.failed?).to      be_falsey
+    expect(status.deleted?).to     be_falsey
+    expect(status.rolled_back?).to be_falsey
+    expect(status.normalized_status).to eq(['create_complete', ''])
+  end
+
+  it 'parses Failed' do
+    miq_task.status = MiqTask::STATUS_ERROR
+    status          = described_class.new(miq_task, nil)
+
+    expect(status.completed?).to   be_truthy
+    expect(status.succeeded?).to   be_falsey
+    expect(status.failed?).to      be_truthy
+    expect(status.deleted?).to     be_falsey
+    expect(status.rolled_back?).to be_falsey
+    expect(status.normalized_status).to eq(['failed', 'Stack creation failed'])
+  end
+
+  # TODO:  remove or implement?  Is canceling something we can handle?
```

Comment:
Left this test in commented out to show what it was testing previously, but I plan on deleting it before a merge.  Curious if canceling a job is even possible with `ansible-runner` and the `MiqQueue` workflow, so knowing that will probably swiftly determine if this makes sense or not.

Reply:
I think you can execute runner in the background and can also stop it https://ansible-runner.readthedocs.io/en/latest/standalone.html#executing-runner-in-the-background

But this can wait for a more targeted fix after this PR I think.

Reply:
Some thing of note to color my comment:

There isn't a `state` or `status` in `MiqTask` that represents "canceled", or at least that I could determine from the constants in that file, so that was mostly where I was pulling my rational for the above.  And I don't know if there is a way within the UI to do something like that for a `MiqTask`/`Job`.

But, I clearly find new things about this code base on a daily basis, so maybe there is.

Reply:
I'm not sure how it's all wired up, but on the tasks page there is a "Cancel Job" button:
![image](https://user-images.githubusercontent.com/12697904/59128202-4d06ab00-8937-11e9-84a8-03a055bb358c.png)

Reply:
Yeah, looks like there is something in `::Job` as well:

https://github.com/ManageIQ/manageiq/blob/09a9f1a07d834008eb9415ff5ce49b4233d28838/app/models/job.rb#L103-L109

Which is what is used by the tower provider as well.  Might still be something we can handle in a follow up then?.  Regardless, thank you for correcting me.

Reply:
I believe the ability to cancel an ansible thing was being added, so we should just be sure we are aligned here.  @lfu @tinaafitz  Do you have more info on this?

Reply:
I don't think it is currently supported to cancel a job on Ansible Tower side after submitting a  job request. Not even sure if Ansible Tower itself supports cancelling a Job once it is launched. 
You may cancel the Miq Job which would abort the process but it won't abort the job on Ansible Tower side if it has been launched.

Reply:
For what it is worth, the repo goes to the trouble of implementing a `Job::Status.canceled?`:

https://github.com/ManageIQ/manageiq-providers-ansible_tower/blob/c43a0e0a9a0834e04b7008ba97732bdfd3d9b9fd/app/models/manageiq/providers/ansible_tower/shared/automation_manager/job/status.rb#L10-L12

so mostly basing my question from that.